src/linter/linter.go

```diff
@@ -92,7 +91,7 @@ func (l *Linter) analyzeFiles(readFileNamesFunc workspace.ReadCallback, allowDis
 
 func (l *Linter) InitStubs(readFileNamesFunc workspace.ReadCallback) {
 	meta.SetLoadingStubs(true)
-	l.analyzeFiles(readFileNamesFunc, nil)
```

Comment:
I'm not sure this is needed.
When Stubs are initialized, we don't need to apply any kind of filters.
Is there a reason behind this line change? It's not related to PHP extentions.